### ISSUE TITLE
Automate announcement/reminder emails

### ISSUE THREAD

**Issue by jamezpolley** [sydney-linux-user-group/slug#4]
Currently, there's one button (publish) which makes an event live AND sends an email at the same time.

It would be useful to disconnect the two.

For example, we'd like to publish all events for the next year in the RSS and iCal feeds, and directly on the website - but only send announcements about each event 30 days prior to the event.

Reminder emails 7 days and 1 day before the event would be nice.

In short: disconnecting the two acts of "publishing" and "announcing" an event. Make announcements schedulable.

**Comment by jamezpolley** [comment]
I envisage 3 different scenarios for future-dated announcements:
- Grab the event details as currently displayed on the website and send them as an email
- Send specific text as an email - for instance, we might want to send an email calling for lightning talks 21 and 14 days before an event, without regard to what the event is currently displaying on the website
- Update the website and feed content and send an email at the same time. For instance, we might want to prepare an announcement announcing a special guest speaker, and have it emailed out as an update at the same time the website and feeds get changed.

**Comment by jamezpolley** [comment]
We've had parts of this since before the last meeting.

"Publishing" to the website and "Announcing" via email are two seperate steps.
